src/plugins/Show.tsx

```diff
@@ -0,0 +1,24 @@
+import * as _ from "lodash";
+import * as React from "react";
+import {homeDirectory, pluralize, resolveDirectory, resolveFile, mapObject} from "../utils/Common";
+import {existsSync, statSync} from "fs";
+import {PluginManager} from "../PluginManager";
+import {Job} from "../shell/Job";
+
+PluginManager.registerOutputDecorator({
+    decorate: (job: Job): React.ReactElement<any> => {
+        const rows = job.screenBuffer.toLines().map(row => {
+            let fullPath: string = "hello";
+
+            fullPath = job.environment.cdpath.map(path => resolveDirectory(path, row))[0].slice(0, -1);
```

Comment:
`CDPATH` shouldn't be used for anything but `cd`. https://www.gnu.org/software/bash/manual/bashref.html#Bourne-Shell-Variables